packages/iTwinUI-react/src/core/Tree/TreeNode.tsx

```diff
@@ -130,10 +125,7 @@ export const TreeNode = (props: TreeNodeProps) => {
   const nodeRef = React.useRef<HTMLLIElement>(null);
 
   const styleDepth = React.useMemo(
-    () =>
-      getWindow()?.CSS?.supports?.(`--level: ${nodeDepth}`)
-        ? { '--level': nodeDepth }
-        : { marginLeft: nodeDepth ? nodeDepth * 28 : 0 },
+    () => ({ '--level': nodeDepth } as React.CSSProperties),
```

Comment:
no need to memo this. i would say move it to inline

Reply:
Fixed